scripts/setup_bls_build_flags.sh

```diff
@@ -1,8 +1,17 @@
 # no shebang; to be sourced from other scripts
 
+unset -v progdir
+case "${0}" in
+*/*) progdir="${0%/*}";;
+*) progdir=.;;
+esac
+
 unset -v gopath
 gopath=$(go env GOPATH)
 HMY_PATH="${gopath%%:*}/src/github.com/harmony-one"
+if [ ! -d $HMY_PATH ]; then
+   HMY_PATH=$(realpath $progdir/../..)
```

Comment:
I think we can just leave this line and remove gopath-based logic as we moved to modules and shouldn't pull in GOPATH anymore.  That is, we can replace lines 9–14 with just this one line.  Existing dev environments already conform to this standard—which essentially says the `bls` and `mcl` repo directories are in the same directory as the `harmony` repo directory—so I think it will be backward compatible.

Reply:
okay.